packages/server/src/routes/members/createMember.ts

```diff
@@ -411,8 +421,59 @@ export const createMember = (
       }
     });
 
+    _addEmailToMailingLists(
+      sgClient,
+      call.body.emailAddress,
```

Comment:
Do we want to add their name too? So we can auto-populate their name when we send?

Reply:
Sure

Reply:
Ah SendGrid separates them into "First Name" and "Last Name" but we only have "Full Name". In other languages like Korean it's all mushed together.

Reply:
I can just save the full name into first name but that could get confusing

Reply:
Yeah people talk about how to deal with different intl naming formats, with basically ending up at "if you can just say, 'what do you want to be called'" that's ideal; in our case we need more than that, some people do 'given name/family name' as a more flexible combo but that's not very clear to users and doesn't cover all cases, so maybe youd have to do different display of that per user. You can also do "full name" and "nickname/what should we call you", where you use "nickname" for the casual feel. Or just stop trying and live with just full name lol

Reply:
Are those fields configurable in sendgrid? should be right?

Reply:
Ah, they do have custom fields. I'll add a full_name for now?